install/aws/aws-cloudwatch-metric-streams/install.yml

```diff
@@ -1,8 +1,8 @@
 id: aws-cloudwatch-metric-streams
-name: AWS-CloudWatch-Metric-Streams
-title: AWS Metric Streams
+name: AWS CloudWatch Metric Streams
+title: AWS CloudWatch Metric Streams
```

Comment:
Double checked this on newrelic.io - Knowing the pre-fix issues and with the upcoming patch for it, i _believe_ this shouldn't cause any issues